docs/src/api/class-elementhandle.md

```diff
@@ -642,6 +642,16 @@ The quality of the image, between 0-100. Not applicable to `png` images.
 Hides default white background and allows capturing screenshots with transparency. Not applicable to `jpeg` images.
 Defaults to `false`.
 
+### option: ElementHandle.screenshot.captureBeyondViewport
```

Comment:
Let's revert it, we'll be using it unconditionally when screenshot goes beyond viewport.